api/project.py

```diff
@@ -41,3 +64,62 @@ def get(self, request) -> JSONResponse:
             "knowledge_base_ids": [str(list.id) for list in project.knowledge_bases],
         }
         return JSONResponse(result)
+
+
+class ProjectCreationFromWorkflow(HTTPEndpoint):
+    async def post(self, request) -> JSONResponse:
+        user_id = request.query_params["user_id"]
+        request_body = await request.json()
+        name = request_body["name"]
+        description = request_body["description"]
+        tokenizer = request_body["tokenizer"]
+        store_id = request_body["store_id"]
+
+        user = auth_manager.get_user_by_id(user_id)
+        organization = auth_manager.get_organization_by_user_id(user.id)
+
+        if not bucket_exists(str(organization.id)):
+            create_bucket(str(organization.id))
+
+        project = project_manager.create_project(
+            organization.id, name, description, user.id
+        )
+        project_manager.update_project(project_id=project.id, tokenizer=tokenizer)
+
+        Session = sessionmaker(get_workflow_db_engine())
+        with Session() as session:
+            results = session.execute(
+                f"SELECT record FROM store_entry WHERE store_id = '{store_id}'"
+            ).all()
+
+        data = [result for result, in results]
```

Comment:
maybe replace var names by more expressive naming

Reply:
good advice, changed it to:
```python
def get_records_from_store(store_id: str) -> List[Dict[str, Any]]:
    Session = sessionmaker(get_workflow_db_engine())
    with Session() as session:
        record_entity_list = session.execute(
            f"SELECT record FROM store_entry WHERE store_id = '{store_id}'"
        ).all()

    record_dict_list = [result for result, in record_entity_list]
    return record_dict_list
```